src/csharpExtensionExports.ts

```diff
@@ -20,4 +22,10 @@ export interface CSharpExtensionExports {
     initializationFinished: () => Promise<void>;
     logDirectory: string;
     profferBrokeredServices: (container: GlobalBrokeredServiceContainer) => void;
+    sendRequest: <Params, Response, Error>(
```

Comment:
Do we also need something for notifications, since I think we eventually learned (the hard way) that a request w/o a response and a notification aren't technically the same thing?

Reply:
I haven't needed it so far. I have no doubt someone might need it, but what I implemented was what I needed.